src/tests/integration/README.md

```diff
@@ -60,7 +60,7 @@ This directory contains template files that run integration tests against the AP
 2. Run the integration test:
 
     ```shell
-    $ python integration-test.py -v --config path/to/configuration.json --openapi path/to/openapi.yaml
+    $ python3 integration-test.py -v --config path/to/configuration.json --openapi path/to/openapi.yaml
```

Comment:
This change should be in the skeleton. I suppose it doesn't hurt to have it here.